server/protocols/jmap-integration-testing/memory-jmap-integration-testing/src/test/resources/listeners.xml

```diff
@@ -28,6 +28,16 @@
       <thresholds>
         <threshold>0.1</threshold>
       </thresholds>
+      <name>first</name>
+    </configuration>
+  </listener>
+  <listener>
+    <class>org.apache.james.mailbox.quota.mailing.listeners.QuotaThresholdCrossingListener</class>
+    <configuration>
+      <thresholds>
+        <threshold>0.1</threshold>
```

Comment:
I propose we do use different thresholds.

Reply:
I thought that using the same threshold make it more clear that we expect the two listeners to be independent. Of course it's a trade off because of the single config file limitation.

Reply:
Having two different listeners allows to ensure one mail come from n°1, the second from n°2